userguide/tasks.rst

```diff
@@ -168,7 +168,12 @@ shows the configuration options for Cloud Syncs.
    | Enabled                | checkbox            | Unset to temporarily disable this Cloud Sync.                                                          |
    |                        |                     |                                                                                                        |
    +------------------------+---------------------+--------------------------------------------------------------------------------------------------------+
-
+   
+   
+.. note:: The time chosen for the Cloud Sync refers to the time that
+   the task is able to start syncing. It does not mean that is when the
```

Comment:
```suggestion
   the task is able to start syncing. It is not when the
```

Reply:
I suspect we would be better off without the second sentence at all.  We tell the reader what it is, so it is not necessary to tell them what it is not.

In the first sentence, it is clearer if s/is able to/is allowed to/